KinoPoiskAutomatedTests/BaseClass/BaseTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using KinoPoiskAutomatedTests.PageObjects;
+using KinoPoiskAutomatedTests.TestData;
+using KinoPoiskAutomatedTests.WebDriver;
+using NUnit.Framework;
+
+namespace KinoPoiskAutomatedTests.BaseClass
+{
+    public class BaseTest 
+    {
+        public TestDetails Data = JsonManager.GetTestData();
+        internal HomePage homePage = new HomePage();
+        internal LoginPage loginPage = new LoginPage();
+        internal PasswordPage passwordPage = new PasswordPage();
+        internal FilmPage filmPage = new FilmPage();
+        internal MyKinopoiskPage myKinopoisk = new MyKinopoiskPage();
+
+        [SetUp]
+        public void Authorization()
+        {
+            homePage.PressLogInButton();
+            loginPage.EnterLoginAndSubmit(Data.Login);
+            passwordPage.EnterPasswordAndEnter(Data.Password);
+        }
+
+        [OneTimeSetUp]
+        public void Open()
+        {
+            var driver = DriverFactory.GetDriver();
```

Comment:
Not a mistake, but I could offer you better implementation.
You could create
`DriverFactory.InitializeDriver()`
method, that would store state of WebDriver in _webDriver.
In the BaseTest class, instead storing of WebDriver in driver variable, you could use property 
protected IWebDriver WebDriver => WebDriverFactory.Driver;

Reply:
resolved